.github/workflows/buf-push.yaml

```diff
@@ -0,0 +1,24 @@
+name: buf (Push)
+on:
+  - push
+jobs:
+  buf:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install buf cli
+        uses: bufbuild/buf-setup-action@v1
+        with:
+          version: 1.32.0-beta.1 # remove this flag when 1.32.0 is released so we go back to always tracking latest.
+                                 # todo: use .bufversion or the env variable?
```

Comment:
this line could be deleted if the action already supported .bufversion, which is in the proposal, but not implemented by the current action